stable/jenkins/templates/jenkins-master-networkpolicy.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.enableNetworkPolicy }}
+kind: NetworkPolicy
+apiVersion: extensions/v1beta1
```

Comment:
Should we variablize the apiVersion so it will work cross release. Like https://github.com/lachie83/charts/blob/master/incubator/istio/templates/rbac.yaml#L4